addons/account/wizard/account_payment_register.py

```diff
@@ -589,6 +594,16 @@ def _onchange_amount(self):
     def _onchange_payment_date(self):
         self.wizard_change_mode = 'payment_date_change'
 
+    @api.onchange('payment_method_line_id')
+    def _onchange_payment_method_line_id(self):
```

Comment:
same comment here as in payments:
we want to limit the use of onchanges and you are adding a cyclic dependency. It's error prone by design :/

Reply:
Same here i could put the code in the compute of journal_id but same that for the other i would have to put payment method line as precompute  :grimacing:

Reply:
Here let's try to use the compute of the journal